runtime/fundamentals/modules.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Using modules in Deno"
+oldUrl:
+  - /runtime/manual/basics/modules/
+  - /runtime/manual/basics/modules/integrity_checking/
+  - /runtime/manual/basics/modules/module_metadata/
+  - /runtime/manual/basics/modules/publishing_modules/
+  - /runtime/manual/basics/modules/reloading_modules/
+  - /runtime/manual/basics/vendoring/
+  - /runtime/manual/advanced/http_imports/
+  - /runtime/manual/advanced/publishing/dnt/
+  - /runtime/manual/advanced/publishing/
+  - /runtime/manual/examples/manage_dependencies
+  - /runtime/manual/node/cdns.md
+  - /runtime/manual/linking_to_external_code
+  - /runtime/manual/linking_to_external_code/reloading_modules
+  - /runtime/fundamentals/esm.sh
+---
+
+Deno uses
+[ECMAScript modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
+as its default module system to align with modern JavaScript standards and to
+promote a more efficient and consistent development experience. It's the
+official standard for JavaScript modules, allows for better tree-shaking,
+improved tooling integration, and native support across different environments.
+
+By adopting ECMAScript modules, Deno ensures compatibility with the evolving
```

Comment:
This sentence is loooooong and a bit unwieldy, I'd suggest splitting it up a bit.

By adopting ECMAScript modules, Deno ensures compatibility with the ever-evolving JavaScript ecosystem. For developers, this means a streamlined and predictable module system that avoids the complexities associated with legacy module formats like CommonJS.

Reply:
Good point, that reads much more natural. Looks like the German came through in me!